proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,175 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only apply for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF/DC wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF/DC could offer
```

Comment:
Also, it is the SCF that are training instructors, we are doing it with the collaboration of DC, but it is only SCF that is actually doing it.